tests/halide.rs

```diff
@@ -322,12 +319,15 @@ mod test {
         let oopsla_halide: Ruleset<Pred> = Ruleset::from_file("baseline/oopsla-halide.rules");
         let oopsla_duration = Duration::from_secs_f32(3.354);
 
+        logger::write_output(&all_rules, &baseline, "halide", "halide", duration, false);
+
         logger::write_output(
             &oopsla_halide,
             &baseline,
             "oopsla halide (1 iter)",
             "halide",
             oopsla_duration,
+            true,
```

Comment:
make this false too

Reply:
oh, why?

Reply:
if we're not running derivability for the enumo halide recipe, there's no point running it for the baseline. i think it'll make it more clear in the nightly report that it just didn't run if everything is 0s